### DISCUSSION TITLE
Add "copy link" action to toolbar settings.

### DISCUSSION THREAD

**Discussion by gety9** [plateaukao/einkbro#280] [General]
I often need to copy url of current page to clipboard, it will be handy if i could simply click icon on toolbar (1 step), instead of clicking menu > copy link (2 steps) or title > copy (2 steps)

Also currently "copy link" icon and "duplicate page" icon are same, maybe it's good idea to change "duplicate page" to https://icon-icons.com/icon/content-duplicate/135741 or smng similar?

**Comment by plateaukao** [comment]
I'll change duplicate icon to a more proper one.

As for copy url feature as an icon, I will consider it in the future.
